test/acceptance/features/bindAppToService.feature

```diff
@@ -269,9 +270,171 @@ Feature: Bind an application to a service
             status:
                 ready: true
             """
-        Then Secret "binding-request-backend" contains "CustomReady" key with value "true"
+        Then Secret should be updated
+        Then Secret contains "CustomReady" key with value "true"
 
 
+    Scenario: Backend Service status update gets propagated to the binding secret for Volume Mount
+        Given OLM Operator "backend" is running
+        * Generic test application "generic-app-demo-volume-mount" is running with binding root as "/var/data"
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo-volume-mount
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/ready: path={.status.ready}
+            spec:
+                host: example.common
+            """
+        * Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-backend-volume-mount
+            spec:
+                bindAsFiles: true
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo-volume-mount
+                    id: SBR
+                customEnvVar:
```

Comment:
this does not exist anymore, got renamed to `mappings`. Please update. Do we need this block at all?